ckanext/canada/plugins.py

```diff
@@ -406,12 +406,16 @@ def dataset_facets(self, facets_dict, package_type):
 
     def group_facets(self, facets_dict, group_type, package_type):
         ''' Update the facets_dict and return it. '''
+        if group_type == 'organization':
```

Comment:
add a comment here with a FIXME linking to the upstream issue, please

Reply:
@wardi comments added: https://github.com/open-data/ckanext-canada/pull/1313/commits/79a72e41f00d2e071b47fdb8d687bdbc9234a2cc

We can just fully remove the `group_facets` methods once it gets patched.